apps/example/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()]
+});
```

Comment:
I see we have some path aliases in `tsconfig.json`. Don't we need to account for it here in `vite.config.ts`

Reply:
I think we can ignore this for now. Since we are not using the the path aliases mentioned `tsconfig.json` in the code

Reply:
okay got it